inc/Engine/Optimization/LazyRenderContent/Frontend/Controller.php

```diff
@@ -181,7 +185,7 @@ private function get_exclusions(): array {
 	 * @return array
 	 */
 	public function add_custom_data( array $data ): array {
-		$data['status']['lrc'] = $this->context->is_allowed();
+		$data['status']['lrc'] = ! rocket_get_constant( 'DONOTROCKETOPTIMIZE' ) && $this->context->is_allowed();
```

Comment:
We don't need the extra check here, we already do that before injecting the beacon [here](https://github.com/wp-media/wp-rocket/blob/00ed622a2f6ac34448de53136ce5df82171dcb8f/inc/Engine/Common/PerformanceHints/Frontend/Processor.php#L99)
```suggestion
		$data['status']['lrc'] = $this->context->is_allowed();
```